docs/docs/deploy/flightcontrol.md

```diff
@@ -4,21 +4,20 @@ description: How to deploy a Redwood app to AWS via Flightcontrol
 
 # Deploy to AWS with Flightcontrol
 
-[Flightcontrol](https://www.flightcontrol.dev?ref=redwood) enables any developer to deploy to AWS without being a wizard. It's extremely easy to use but lets you pop the hood and leverage the raw power of AWS when needed. It supports servers, static sites, and databases which makes it a perfect fit for hosting scalable Redwood apps.
+[Flightcontrol](https://www.flightcontrol.dev) enables any developer to deploy to AWS without being a wizard. It's extremely easy to use but lets you pop the hood and leverage the raw power of AWS when needed. It supports servers, static sites, and databases which makes it a perfect fit for hosting scalable Redwood apps.
```

Comment:
I'd leave the `ref` in there. IIRC someone from flightcontrol first wrote these docs, so they probably want that to better know where their traffic is coming from 
```suggestion
[Flightcontrol](https://www.flightcontrol.dev?ref=redwoodv) enables any developer to deploy to AWS without being a wizard. It's extremely easy to use but lets you pop the hood and leverage the raw power of AWS when needed. It supports servers, static sites, and databases which makes it a perfect fit for hosting scalable Redwood apps.
```